doc/man3/EVP_RAND.pod

```diff
@@ -62,7 +62,7 @@ EVP_RAND_STATE_ERROR - EVP RAND routines
                      const unsigned char *ent, size_t ent_len,
                      const unsigned char *addin, size_t addin_len);
  int EVP_RAND_nonce(EVP_RAND_CTX *ctx, unsigned char *out, size_t outlen);
- int EVP_RAND_enable_locking(EVP_RAND_CTX *ctx);
+ int EVP_RAND_enable_locking(EVP_RAND_CTX *rand);
```

Comment:
Why change from _ctx_ to _rand_ for just this one call?
The remaining calls still use _ctx_.

Reply:
All the changes I made are based on their defination in source code. For example, this one is defined as **int EVP_RAND_enable_locking(EVP_RAND_CTX *rand)** in crypto/evp/evp_rand.c. IMO if ctx is more readable than rand, source code is better to keep ctx too.

Reply:
That's fine if you think it's not a deal. I will close this pr and only keep the change like miss **const** keyword and etc.

Reply:
That's fine if you think it's not a deal. I will close this pr and only keep the change like miss **const** keyword and etc.